index.bs

```diff
@@ -2837,7 +2836,7 @@ Note: The {{CollectedClientData}} may be extended in the future. Therefore it's
 
         <div dfn-type="dict-member" dfn-for="TokenBinding">
             :   <dfn>status</dfn>
-            ::  This member is one of the following:
+            ::  This member SHOULD be a member of {{TokenBindingStatus}} but [=client platforms=] MUST ignore unknown values, treating an unknown value as if the {{CollectedClientData/tokenBinding}} member was absent. When known, this member is one of the following:
```

Comment:
This is a required member so I think the client would need to treat unknown values as if the whole `TokenBinding` structure was absent.

Reply:
That's what this says, is it not?

Reply:
Yes, sorry.